.vaunt/config.yaml

```diff
@@ -0,0 +1,174 @@
+version: 0.0.1
+actions:
+    - action:
+          type: point
+          name: docs_dash_pr_submitted
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in {'2024-tidb-docs-dash','contribution'}"
+                    value: 4
+    - action:
+          type: point
+          name: docs_dash_pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
+                        & labels in {'2024-tidb-docs-dash','contribution'}"
+                    value: 10
+    - action:
+          type: point
+          name: docs_dash_issue_contribution
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in {'2024-tidb-docs-dash'}
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']"
+                    value: 2
+    - action:
+          type: point
+          name: issue_contribution
```

Comment:
@elewis787 will this only work when someone comments on an existing issue? Or we need to create a separate action for when someone creates a new issue during the event?

Reply:
Actually looking at [this](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment), we maybe covered.

Reply:
You do not need to create a separate issue. This will pick up comments on any issue as long as the comment is created between the specified dates in the condition.

Reply:
@jeff1010322 sorry if I wasn't clear. We also want to award a point to people when they open a new issue. 

For example, someone may find a typo in a documentation page, and they can open a new issue to report a "bug". We want to award this person one point after the reviewers accept the contribution with a thumbs up emoji.

Reply:
We can create a new point for issue actions. 

Do you want this one only for issues that get labeled docs dash or for any issue opened that then receives a thumbs up from the moderators? 

Will the point value be 1?

Reply:
Hi @elewis787 yes, the point value should be 1. It can be for any new issue that's opened between 01-09-2024 and 01-12-2024 (with the same timestamp as the rest).

Reply:
I have updated the PR to include the new point.

Reply:
Hi @elewis787, may I know whether you have submitted the new updates to the commit history https://github.com/pingcap/docs/pull/15947/commits?

Reply:
sorry, I didn't update my remote branch - the commit is there now.

Reply:
https://github.com/pingcap/docs/pull/15947/commits/e3498c0897ff07259fbbc46d599ab6bcf517d4a3